django/core/secret_key.py

```diff
@@ -0,0 +1,39 @@
+from django.conf import settings
+from django.utils.module_loading import import_string
+
+
+class SecretKeySettingsBackend:
```

Comment:
`SettingsSecretKeys` a better name, other backends can be called e.g. `DatabaseSecretKeys` ?

Reply:
Not relevant anymore since this PR no longer contains the settings backend stuff. :)